src/main/java/com/powsybl/openloadflow/ac/DistributedSlackOnLoadOuterLoop.java

```diff
@@ -38,16 +40,21 @@ public String getType() {
     @Override
     public List<ParticipatingElement<LfBus>> getParticipatingElements(LfNetwork network) {
         return network.getBuses()
-            .stream()
-            .filter(bus -> bus.getPositiveLoadCount() > 0 && getVariableLoadTargetP(bus) > 0)
-            .map(bus -> new ParticipatingElement<>(bus, getVariableLoadTargetP(bus)))
-            .collect(Collectors.toList());
+                .stream()
+                .filter(bus -> bus.getPositiveLoadCount() > 0 && getVariableLoadTargetP(bus) > 0)
+                .map(bus -> new ParticipatingElement<>(bus, getVariableLoadTargetP(bus)))
+                .collect(Collectors.toList());
     }
 
     private double getVariableLoadTargetP(LfBus bus) {
         return distributedSlackOnConformLoad ? bus.getLoadTargetP() - bus.getFixedLoadTargetP() : bus.getLoadTargetP();
     }
 
+    private double getVariableLoadTargetQAgainstVariableLoadTargetP(LfBus bus) {
+        return distributedSlackOnConformLoad ? (bus.getLoadTargetQ() - bus.getFixedLoadTargetQ()) / (bus.getLoadTargetP() - bus.getFixedLoadTargetP())
```

Comment:
Maybe check that bus.getLoadTargetP() != bus.getFixedLoadTargetP() to avoid the division by 0.

Reply:
Don't know why, but it seems we cannot get division by 0 in reality (tested with recollement-auto-20190124-2110-enrichi.xiidm) ?!?

However we can fix this in getVariableLoadTargetQAgainstVariableLoadTargetP with :
- throwing a ComputationException ?
- or returning 0 ?

Reply:
Indeed you check before that the variable part is greater than zero to participate.